web/src/components/DandisetsPage.vue

```diff
@@ -117,7 +136,7 @@ export default {
       return { page, sortOption, sortDir };
     },
     dandisets() {
-      return this.dandisetRequest ? this.dandisetRequest.data : [];
+      return this.dandisetRequest ? this.dandisetRequest.data : undefined;
```

Comment:
Why was this changed?

Reply:
To enable the loading icon to work properly. `undefined` means that no request has been made yet, so it should display the spinner. `[]` means that a request has been made and there's nothing to show, so the `no-content` slot is rendered. Anything else means there's dandisets, so it renders as it used to.

Reply:
Gotcha